.npmignore

```diff
@@ -0,0 +1,13 @@
+.circleci
```

Comment:
Generally it's probably better to use the `files` property in `package.json` instead of `.npmignore`. It avoids hazards [like this](https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d), and `npm` have a sort of... soft recommendation to prefer `files` in their docs as well.

Reply:
Yeah as I was updating it, I was thinking of this